catalogue_pipeline/matcher/src/main/scala/uk/ac/wellcome/platform/matcher/models/LinkedWorksIdentifiersList.scala

```diff
@@ -1,3 +1,5 @@
 package uk.ac.wellcome.platform.matcher.models
 
-case class LinkedWorksIdentifiersList(linkedWorks: Set[IdentifierList])
+import uk.ac.wellcome.models.matcher.MatchedIdentifiers
+
+case class LinkedWorksIdentifiersList(linkedWorks: Set[MatchedIdentifiers])
```

Comment:
`linkedWorks`: should this change too?

Reply:
yep, that's in the PR which ditches LinkedWorksIdentifiersList